StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1010CodeFixProvider.cs

```diff
@@ -64,26 +65,25 @@ private static Task<Document> GetTransformedDocumentAsync(Document document, Syn
                 SyntaxToken precedingToken = token.GetPreviousToken();
                 if (precedingToken.TrailingTrivia.Any(SyntaxKind.WhitespaceTrivia))
                 {
-                    SyntaxToken corrected = precedingToken.WithoutTrailingWhitespace().WithoutFormatting();
-                    replacements[precedingToken] = corrected;
+                    replacements.Add(precedingToken);
                 }
             }
 
-            if (!token.TrailingTrivia.Any(SyntaxKind.EndOfLineTrivia) && token.TrailingTrivia.Any(SyntaxKind.WhitespaceTrivia))
+            SyntaxTriviaList trailingTrivia = token.TrailingTrivia;
+            if (!trailingTrivia.Any(SyntaxKind.EndOfLineTrivia) && trailingTrivia.Any(SyntaxKind.WhitespaceTrivia))
             {
-                SyntaxToken corrected = token.WithoutTrailingWhitespace().WithoutFormatting();
-                replacements[token] = corrected;
+                replacements.Add(token);
             }
 
             if (replacements.Count == 0)
             {
-                return Task.FromResult(document);
+                return document;
             }
 
-            var transformed = root.ReplaceTokens(replacements.Keys, (original, maybeRewritten) => replacements[original]);
+            var transformed = root.ReplaceTokens(replacements, (original, maybeRewritten) => original.WithoutTrailingWhitespace().WithoutFormatting());
```

Comment:
:bulb: I can't see what you changed because you squashed the commits. In the future, try to avoid squashing commits created during the code review process except in cases where the primary reviewer requests it.